app_dart/lib/src/request_handlers/scheduler/vacuum_stale_tasks.dart

```diff
@@ -54,6 +54,9 @@ class VacuumStaleTasks extends RequestHandler<Body> {
   Future<void> _vacuumRepository(gh.RepositorySlug slug) async {
     final DatastoreService datastore = datastoreProvider(config.db);
 
+    final FirestoreService firestoreService = await config.createFirestoreService();
+
+    // Datastore logic to query for recent tasks and assign to commits
     final List<FullTask> tasks = await datastore.queryRecentTasks(slug: slug).toList();
```

Comment:
Let's use the default `backfillerTargetLimit` for the commit limit: https://github.com/flutter/cocoon/blob/main/app_dart/lib/src/service/config.dart#L173